### PR TITLE
doc: Increment in doc with flutterando_analysis

### PR DESCRIPTION
**Pull request by vitoria-soares**
-  Increment in doc with flutterando_analysis

### CODE REVIEW

**Comment by vitoria-soares** [comment]
- Increment in documentation with package flutterando_analysis

File: lib/src/core/types/either.dart [hunk 1]
```diff
@@ -1,30 +1,54 @@
...
   bool get isLeft;
+
+  /// The get [isRight] it's the type boolean
   bool get isRight;
 
+  ///The method [fold] it's the type [T] generics and recive for params
+  ///the functions [leftFn] and [rightFn].
   T fold<T>(T Function(TLeft l) leftFn, T Function(TRight r) rightFn);
 
+  ///The method [getOrElse] it's the type [TRight] and recive for params
```
Comment (Bwolfs2) [inline lib/src/core/types/either.dart]: I guess is "receive" instead "recive"?

File: lib/src/domain/errors/errors.dart [hunk 2]
```diff
@@ -8,12 +8,22 @@ import 'package:uno/uno.dart';
 ///the [stackTrace] is different from null, if it is, returns it as a string
 ///otherwise, returns an empty string
 class UnoError<TData> implements Exception {
+  ///The varieble [message] it's the type String
   final String message;
+
+  ///The varieble [data] it's the type TData
   final TData? data;
+
+  ///The varieble [stackTrace] it's the type StackTrace
```
Comment (Bwolfs2) [inline lib/src/domain/errors/errors.dart]: "variable" instead  "varieble"

**Review by Bwolfs2 — CHANGES_REQUESTED** [review]

File: lib/src/core/types/either.dart (hunk 1 above)
Comment (vitoria-soares) [reply lib/src/core/types/either.dart]: Correction done

File: lib/src/domain/errors/errors.dart (hunk 2 above)
Comment (vitoria-soares) [reply lib/src/domain/errors/errors.dart]: Correction done

**Review by Bwolfs2 — APPROVED** [review]

**Pull request merged**